network/network_windows.go

```diff
@@ -55,6 +58,17 @@ func UseHnsV2(netNs string) (bool, error) {
 	return useHnsV2, err
 }
 
+// this hnsv2 variable is package level variable in network
+// we do this to avoid passing around os specific objects in platform agnostic code
+var hnsv2 hnswrapper.HnsV2WrapperInterface = hnswrapper.Hnsv2wrapper{}
+
+func enableHnsTimeout(timeoutValue int) {
+	if _, ok := hnsv2.(hnswrapper.Hnsv2wrapperwithtimeout); !ok {
+		var timeoutDuration = time.Duration(timeoutValue) * time.Second
```

Comment:
should I return a error if timeoutvalue is zero or default to to a constant we define? @tamilmani1989 what do you think?